www/js/bear.js

```diff
@@ -49,13 +49,14 @@ angular.module('emission.main.bear',['nvd3', 'emission.services', 'ionic-datepic
           response.otherBears[key]['size'] = response.otherBears[key]['size'] * scale;
           $scope.otherBears.push(response.otherBears[key]);
         }
+        $state.reload();
```

Comment:
Typical fix is to put the updates inside `$scope.$apply`, not reload the page. Have you tested the change?
http://jimhoskins.com/2012/12/17/angularjs-and-apply.html